### PR TITLE
[1-0002: Create infrastructure for project] add HelloController

### PR DESCRIPTION
**Pull request by Saparmyrat12**

### CODE REVIEW

File: epam-vote-system-controller/src/main/java/com/epam/vote/controller/HelloController.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+ */
+
+@RestController
+@RequestMapping("/api")
+public class HelloController {
+    @RequestMapping(method = RequestMethod.GET)
+    public String printHello(){
+        return "Hello world";
+    }
+}
```
Comment (AzarenkaA) [inline epam-vote-system-controller/src/main/java/com/epam/vote/controller/HelloController.java:23]: add empty line

File: epam-vote-system-controller/src/main/java/com/epam/vote/controller/HelloController.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
+package com.epam.vote.controller;
+
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+/**
+ * <p/>
```
Comment (AzarenkaA) [inline epam-vote-system-controller/src/main/java/com/epam/vote/controller/HelloController.java:8]: Add comment

File: epam-vote-system-controller/src/main/java/com/epam/vote/controller/HelloController.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+ * Copyright (C) 2021
+ * <p/>
+ * Date: май 04, 2021
+ *
+ * @author Sapar
+ */
+
```
Comment (AzarenkaA) [inline epam-vote-system-controller/src/main/java/com/epam/vote/controller/HelloController.java:15]: remove empty line

File: epam-vote-system-controller/src/main/webapp/WEB-INF/web.xml [hunk 4]
```diff
@@ -4,4 +4,31 @@
...
+        <listener-class>org.springframework.web.context.ContextLoaderListener</listener-class>
+    </listener>
+
+
+    <servlet>
+        <servlet-name>appServlet</servlet-name>
+        <servlet-class>org.springframework.web.servlet.DispatcherServlet</servlet-class>
+        <init-param>
+            <param-name>contextConfigLocation</param-name>
+            <param-value>classpath:com.epam.voice.system.controller/spring/spring-mvc.xml</param-value>
```
Comment (AzarenkaA) [inline epam-vote-system-controller/src/main/webapp/WEB-INF/web.xml:22]: rename this package name to "vote"

File: epam-vote-system-controller/src/main/java/com/epam/vote/controller/HelloController.java (hunk 2 above)
Comment (Saparmyrat12) [reply epam-vote-system-controller/src/main/java/com/epam/vote/controller/HelloController.java:8]: add comment for import?

File: epam-vote-system-controller/src/main/java/com/epam/vote/controller/HelloController.java [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+import org.springframework.web.bind.annotation.RestController;
+
+/**
+ * This class print Hello world by url /api.
+ * <p/>
+ * Copyright (C) 2021
+ * <p/>
+ * Date: май 04, 2021
+ * @author Sapar.
+ * This class print Hello world by url /api
```
Comment (AntonAzarenka) [inline epam-vote-system-controller/src/main/java/com/epam/vote/controller/HelloController.java]: remove it.

**Pull request merged**